lisp/plugins/list_layout/layout.py

```diff
@@ -251,6 +251,10 @@ def _key_pressed(self, event):
             if QKeySequence(keys) in self._go_key_sequence:
                 event.accept()
                 self.__go_slot()
+            elif event.key() == Qt.Key_Delete:
```

Comment:
This will use the platform/desktop default keybindings
```suggestion
            elif sequence == QKeySequence.Delete:
```